logging/google/cloud/logging/client.py

```diff
@@ -71,11 +71,23 @@ class Client(JSONClient):
     :param http: An optional HTTP object to make requests. If not passed, an
                  ``http`` object is created that is bound to the
                  ``credentials`` for the current object.
+
+    :type use_gax: bool or :class:`NoneType`
+    :param use_gax: An optional parameter that explicitly specifies whether
```

Comment:
s/"An optional parameter that explicitly specifies whether"/"(Optional) Explicitly specifies whether" and then at the end of the description (i.e. after "or HTTP"), say "or HTTP. If unset, falls back to the `FOOBAR` environment variable."